target/linux/kirkwood/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 ARCH:=arm
 BOARD:=kirkwood
 BOARDNAME:=Marvell Kirkwood
-FEATURES:=usb nand squashfs ramdisk
+FEATURES:=usb nand squashfs ramdisk ubifs
```

Comment:
Drop the ubifs feature or provide an explanation why it is required.

Reply:
@braian87b afaik the "ubifs" feature here means you want to generate a single read/write rootfs with ubifs filesystem. The "squashfs" option means it will generate a firmware with a read-only squashfs partition and a read/write ubifs partition.

a single read-write rootfs is not a good idea in most cases, as you will have no failsafe mode or "reset to default" (factory reset). Here on Kirkwoods there is abundant flash, using a few MB for a read-only squashfs root is better.

My opinion is that this "ubifs" option should NOT be added to kirkwood devices (or any device) supported.

Reply:
Sorry but I did not write any of this code, just forked another repo https://github.com/lzto/lede-source/tree/pogoplug-v4 and tried to push in to LEDE (in OpenWrt this device was supported but not on LEDE) and I don't know how to modify it in the way you describe. OpenWrt for this device it uses ubifs too.